app/scripts/controllers/modals/setHomePageModal.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+/* jshint unused: false */
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:SetHomePageModalController
+ * @description
+ * # SetHomePageModalController
+ * Controller of the openshiftConsole
+ */
```

Comment:
I don't think this comment is helpful. I understand a lot of our files have it, but I'd just take it out.